src/LinearFractional.jl

```diff
@@ -189,7 +189,7 @@ end
 function JuMP.delete(model::LinearFractionalModel, vref::LinearFractionalVariableRef)
     JuMP.delete(model.model, vref.vref)
 end
-function JuMP.is_valid(model::LinearFractionalModel, vref::LinearFractionalVariableRef)
+function JuMP.is_valid(model::Union{LinearFractionalModel, JuMP.Model}, vref::LinearFractionalVariableRef)
```

Comment:
This seems like type piracy because we don't want to define it for `JuMP.Model`.

It also looks like it would be a circular call... does the line `            is_valid(model.model, vref.vref))` below cause this to call itself?

Reply:
You are right.I removed the new is_valid method and my test scenario still worked.

Reply:
This is not type piracy because you own `LinearFractionalVariableRef`.

Note that the `InvalidVariableRef` is just a printing thing. So this PR doesn't "fix" anything. You're still missing a way to call `JuMP.is_valid`.